wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController.swift

```diff
@@ -376,7 +387,7 @@ final class ConversationListViewController: UIViewController {
     func applyColorTheme() {
         view.backgroundColor = mainSplitViewState == .expanded
         ? ColorTheme.Backgrounds.backgroundVariant
-        : ColorTheme.Backgrounds.surfaceVariant
+        : ColorTheme.Backgrounds.surface
```

Comment:
Does this change match the Figma design?

Reply:
Yes!

Reply:
Coming from Wolfgang:

> We should to make use of the semantic color variable “Surface” (white on light mode, gray-95 on dark mode) if possible. For background color.